Assets/Scripts/QuestEditor/EditorSelectionList.cs

```diff
@@ -83,7 +84,7 @@ public void SelectItem(UnityEngine.Events.UnityAction call)
         float hOffset = 22;
         foreach (string s in traits)
         {
-            db = new DialogBox(Vector2.zero, new Vector2(10, 1), s);
+            db = new DialogBox(Vector2.zero, new Vector2(10, 1), new StringKey(s,false));
```

Comment:
for this commit all these magic keys will remain same, I will find a solution for the Localization Quest Commit